common/metrics/defs.go

```diff
@@ -794,6 +806,12 @@ const (
 	StandbyTransferTaskQueueLatency
 	StandbyTimerTaskQueueLatency
 	CompleteTaskFailedCounter
+	HistoryCacheRequests
+	HistoryCacheFailures
+	HistoryCacheLatency
+	CacheMissCounter
```

Comment:
Will it be better named as HistoryCacheMissCounter and "history-cache.miss"?

Reply:
Leaving it as-is.  Will address this once we have use case for cache-miss counter which needs to be used by other components.